node/rpc/src/data_types.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use async_std::channel::Sender;
+use async_std::sync::{Arc, RwLock};
+use jsonrpc_v2::{MapRouter as JsonRpcMapRouter, Server as JsonRpcServer};
+use serde::Serialize;
+
+use beacon::{Beacon, BeaconSchedule};
+use blocks::Tipset;
+use blockstore::BlockStore;
+use chain::headchange_json::HeadChangeJson;
+use chain::ChainStore;
+use chain_sync::{BadBlockCache, SyncState};
+use forest_libp2p::NetworkMessage;
+use message_pool::{MessagePool, MpoolRpcProvider};
+use state_manager::StateManager;
+use wallet::KeyStore;
+
+pub type SubscriptionHeadChange = (i64, HeadChangeJson);
+
+#[derive(Serialize)]
+pub struct StreamingData<'a> {
+    pub json_rpc: &'a str,
+    pub method: &'a str,
+    pub params: SubscriptionHeadChange,
```

Comment:
This is what it should be
```suggestion
    pub params: (i64, Vec<HeadChangeJson>),
```

Reply:
That's what was causing the deserialization error earlier, which is why I changed it. I'll put it back, and try to solve the deserialization issue a different way.

Reply:
Deserialization error? I have this working right now... Didn't change much to get this working.

Reply:
Yeah, no, it wasn't difficult to solve, now that I knew where to look. Try now.